src/auth/components/AuthControls.jsx

```
@@ -6,6 +6,7 @@ import {
 } from '../auth-events-schema.js';
 import { dispatchCommand } from '../../shared/events/index.js';
 import { useI18n } from '../../shared/i18n/index.js';
+import { getLoggedInUserProfile } from '../../stores/userProfileStore.js';
```

ESLint

[error] There is no rule allowing dependencies from elements of type "auth" to elements of type "stores"

(boundaries/dependencies)